deploy/kubernetes/base/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: lustre-csi-system
+images:
```

Comment:
Put 'images:' at the end of the file.

The `kustomize edit` will modify this file to set the image name and tag, and in the past we've sometimes seen that it can mangle anything that follows the 'image' section.

Reply:
Done